content/integrations/kubernetes.md

```diff
@@ -120,6 +120,48 @@ instances:
 
 See the [example kubernetes.yaml](https://github.com/DataDog/integrations-core/blob/master/kubernetes/conf.yaml.example) for all available configuration options.
 
+#### Custom integration
```

Comment:
Do we want to recommend this vs annotations?

Reply:
Good point, I'd personally think we could document both.
From my experience however, configmaps are used to configure integrations and annotations are used for autodiscovery.